deploy/deploy.sh

```diff
@@ -29,12 +29,15 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 $DIR/cleanup.sh
 
-$PGO_CMD get clusterrole pgo-cluster-role 2> /dev/null > /dev/null
-if [ $? -ne 0 ]
+if [[ "${PGO_NAMESPACE_MODE:-dynamic}" != "disabled" ]]	
```

Comment:
Hm, I feel like the default here contrasts with the [ansible rule](https://github.com/CrunchyData/postgres-operator/pull/1396/files#diff-8b078c64bd0ddf6cc8b7e1e3d60c147dR98) which is `disabled`.

I think we need to default to one or the other.

I'm torn between `dynamic` and `disabled` -- I think `disabled` is a lower barrier to entry though esp. if one does not have permission to create `ClusterRole`

Reply:
The default for Ansible is also `dynamic` (see file `installers/ansible/roles/pgo-operator/defaults/main.yml`).

I do see the potential confusion however, since the `inventory` has commented out variable that specifies `disabled`, e.g.:
```
# namespace_mode='disabled'  
```
I did this to have an easy solution for disabling (i.e. just remove the `#`), but I can def change either of these (the default and `inventory` var) as needed.

Reply:
I would just do

```ini
# namespace_mode='dynamic'
```

Really the commented out ones should mirror the defaults.

Reply:
`dynamic` is now specified.